src/timeline/js/controllers.js

```diff
@@ -1150,7 +1150,7 @@ $scope.SetTrackLabel = function (label){
 			var abs_diff = Math.abs(diff);
 			
 			// Check if this clip is nearby
-			if (abs_diff < smallest_abs_diff && abs_diff <= threashold) {
+			if (abs_diff < smallest_abs_diff && abs_diff <= threshold) {
```

Comment:
source change, please review

Reply:
Corresponding variable invocation to previous.